astropy/time/formats.py

```diff
@@ -1261,6 +1336,58 @@ def set_jds(self, val1, val2):
                               *iterator.operands[1:])
         self.jd1, self.jd2 = day_frac(jd1, jd2)
```

Comment:
Concretely, the suggestion would be to `return day_frac(jd1, jd2)` here

Reply:
Done